qt/widgets/common/CMakeLists.txt

```diff
@@ -1,3 +1,7 @@
+# Add the Python scripts directory to the build
+set(PYTHON_SCRIPTS_DIR ${CMAKE_SOURCE_DIR}/mantid/qt/python/mantidqt/mantidqt/widgets/helpwindow)
```

Comment:
This should be done in the pyproject.toml or somewhere similar. I don't think that it will work once packaged.